evil-colemak-basics.el

```diff
@@ -118,6 +128,14 @@ rotated; see evil-colemak-basics-rotate-t-f-j."
           "j" 'evil-snipe-j
           "J" 'evil-snipe-J))
        (t (user-error "Invalid evil-colemak-basics-char-jump-commands configuration"))))
+    (when (eq evil-colemak-basics-layout-mod 'mod-dh)
+      (evil-define-key '(motion normal visual) keymap
+	"m" 'evil-next-line)
```

Comment:
something is wrong with indentation here. mixing spaces and tabs it seems. can you fix that? (hint: try `aggressive-indent-mode`)

Reply:
Done. (Somehow I didn't have `indent-tabs-mode` set to nil.)